tests/e2e/080_vrf_test.go

```diff
@@ -193,11 +210,11 @@ func TestVRFsWithSameSubnets(t *testing.T) {
 	Expect(ctx.GetValueState(vppVrf1)).To(Equal(kvscheduler.ValueState_CONFIGURED))
 	Expect(ctx.GetValueState(vppVrf2)).To(Equal(kvscheduler.ValueState_CONFIGURED))
 
-	// try to ping in both VRFs
-	Expect(ctx.PingFromMs(msName, vrfVppIP, PingWithSourceInterface(vrf1Label+tapNameSuffix))).To(Succeed())
```

Comment:
Any particular reason for removing this ping test here?

Reply:
No, thanks for noticing it, those lines were removed by accident.